enterprise/frontend/src/metabase-enterprise/auth/index.js

```diff
@@ -29,14 +29,14 @@ PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections =>
       description: null,
       noHeader: true,
       widget: SamlAuthCard,
-      getHidden: () => !hasPremiumFeature("sso"),
+      getHidden: () => !hasPremiumFeature("sso_saml"),
```

Comment:
I think that #32399 completely removed `sso` flag.
Not sure if we need it as the fallback for downgrades? cc @NevRA 

Also, please note that all other feature flags are kebab-cased but these specific flags are set as the snake_case in #32399.

tl;dr
Not sure if this particular fix is even needed on the FE, or do we want to being back `sso` and to adjust granular flags to be kebab-cased.

cc @ranquild